### ISSUE TITLE
integrity check and follow up actions

### ISSUE THREAD

**Issue by ilippert** [JabRef/jabref#11419]
**Is your suggestion for improvement related to a problem? Please describe.**
I like the integrity check. I am frustrated that I cannot find wizards that help me to solve the errors found.

**Describe the solution you'd like**
I would like to select a finding/warning ([or several at once](https://github.com/JabRef/jabref/issues/11418)) and then use a right click to select a possible action. E.g. for a file name with an `?` offer to repair that, of for a Warning about a latex command (e.g. `\enquote`) I could "register" the command so that it does not generate a warning anymore.

**Comment by HoussemNasri** [comment]
Additional fixable finding/warning/reports can include:
- Citation key deviates from generated key --> generate citation key for that entry
- Capital letters are not masked using curly brackets {} --> Basic string manipulation

We can go with the right click approach or an approach similar to the image below or a combination or both: right-click when multiple possible fixes, and single-click when one possible fix (common case).

![Image](https://github.com/user-attachments/assets/9c025a0d-3485-488a-b1c5-828df32cd08f)


## Random thoughts
- Applied fixes should be transparent to users. This means users should be informed about the previous inconsistent value of a field and the new consistent value that will be applied as a fix, possibly showing a difference (diff).
- Sometimes multiple fixes are possible (.e.g. non-ASCII encoded character found), here we can delete all non-ascii characters, enclosing them in braces, or other actions. The UI should offer multiple choices of fixes when available.
- While not directly related to this feature, integrity checks can sometimes be ignorable. For instance, if I intentionally use non-ASCII characters in a field, I wouldn't want it flagged as inconsistent every time I scan. There should be a column to mark integrity checks as ignored.

**Comment by HoussemNasri** [comment]
Hello @priyanshu16095, you currently have four assigned issues. Please ensure those are completed or unassigned before taking on additional tasks.

**Comment by HoussemNasri** [comment]
Yes, choose one to work on, and unassign yourself from all the other ones.

**Comment by ThiloteE** [comment]
I think @HoussemNasri's [proposal](https://github.com/JabRef/jabref/issues/11419#issuecomment-2411710793) is good. Have a button that says "fix" or "reset" or something like that, will immediately signal users that those problems can be fixed, whereas "rightclick > ..." is a little hidden.
